docs: add module documentation for Registry Cache (#120)
kyma-project/registry-cache: docs/user/troubleshooting/README.md

# Troubleshooting

> Registry Cache configuration is validated before being applied to the cluster. Invalid configuration will be rejected by the webhook.

Reviewer: ```suggestion
Registry Cache configuration is validated before being applied to the cluster. The webhook rejects an invalid configuration.
```

If the configuration is valid but the Registry Cache setup fails on the Kyma Control Plane (KCP) side, the `RegistryCacheConfig` resource status transitions to `Error` with an error message in the status conditions. In this case, contact the Kyma support team for assistance.

> For specific issues, see the guides below.

Reviewer: ```suggestion
For specific issues, see the relevant guides.
```
